src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc6573/invalidate.cache.with.multiple.link.target/accept/client.rpt

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property response1Date ${http_cache:date()} # sharedProperty
+property response2Date ${http_cache:datePlus(5)} # sharedProperty
+
+connect await ROUTED_PROXY
+        "nukleus://streams/http-cache#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/items?query")
+                              .header(":authority", "localhost:8080")
+                              .build()}
+connected
+
+write close
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "200")
+                             .header("content-length", "11")
+                             .header("date", response2Date)
+                             .header("cache-control", "max-age=10")
+                             .header("etag", "first")
+                             .build()}
+
+read "response #1"
+
+read closed
+
+read notify FIRST_REQUEST_COMPLETED
+
+connect await FIRST_REQUEST_COMPLETED
+        "nukleus://streams/http-cache#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "POST")
+                              .header(":path", "/items/1")
+                              .header(":authority", "localhost:8080")
+                              .header("content-length", "9")
+                              .build()}
+connected
+
+write "post body"
+
+write close
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "200")
+                             .header("content-length", "11")
+                             .header("date", response2Date)
+                             .header("link", "</items>; rel=\"collection\", </items>; rel=\"collection\"")
```

Comment:
These `collection` links probably need to be different and each one cached up front so we can ensure neither is being ignored when processed.

Reply:
yep sounds good